xml/System.Diagnostics.Metrics/Meter.xml

```diff
@@ -189,13 +189,13 @@ Example uses for Histogram: the request duration and the size of the response pa
         <Parameter Name="description" Type="System.String" />
       </Parameters>
       <Docs>
-        <typeparam name="T">To be added.</typeparam>
+        <typeparam name="T">The numerical type of the measurement.</typeparam>
         <param name="name">The instrument name. cannot be <see langword="null" />.</param>
         <param name="observeValues">The callback to call to get the measurements when the <see cref="M:System.Diagnostics.Metrics.ObservableCounter`1.Observe" /> is called by <see cref="M:System.Diagnostics.Metrics.MeterListener.RecordObservableInstruments" />.</param>
         <param name="unit">Optional instrument unit of measurements.</param>
         <param name="description">Optional instrument description.</param>
         <summary>ObservableCounter is an Instrument which reports monotonically increasing value(s) when the instrument is being observed.</summary>
-        <returns>To be added.</returns>
+        <returns>Returns a new ObservableCounter.</returns>
```

Comment:
It usually looks nicer to prefer plain text. Camel case is preferred if you use the full `<see cref=""/>` of the API.
```suggestion
        <returns>A new observable counter.</returns>
```